lib/sdotypes.c

```diff
@@ -2429,20 +2399,53 @@ void sdo_rendezvous_free(sdo_rendezvous_t *rv)
 		sdo_string_free(rv->wsp);
 
 	if (rv->me != NULL)
-		sdo_string_free(rv->me);
+		sdo_free(rv->me);
 
 	if (rv->pr != NULL)
-		sdo_string_free(rv->pr);
+		sdo_free(rv->pr);
 
 	if (rv->delaysec != NULL)
 		sdo_free(rv->delaysec);
 
+	if (rv->bypass != NULL)
+		sdo_free(rv->bypass);
+
 	sdo_free(rv);
 }
 
-/**
- * TO-DO : Need to loop through RV instructions.
- * 
+/*
+ * This is a lookup on all possible RVVariable
+ */
+#define BADKEY -1
```

Comment:
Should these definitions be moved to sdotypes.h

Reply:
Done.